doc/sonic-vrf-hld-v1.0.md

```diff
@@ -545,22 +588,19 @@ $ config vrf add <vrf_name>
 $ config vrf del <vrf_name>
 
 //bind an interface to a VRF
-$ config interface <interface_name> vrf bind <vrf_name>
+$ config interface bind <interface_name> <vrf_name>
```

Comment:
My suggestion is to have it as  `config interface vrf add <interface_name> <vrf_name>`. When user just give "`config interface`" the options are listed as `ip, pfc` etc and "`bind`" may not be good choice whereas "`vrf`" can be self explanatory.

Reply:
Updated CLI in HLD